app/[locale]/layout.tsx

```
@@ -0,0 +1,135 @@
+  params: { locale: string };
+}) {
+  const messages = await getMessages({ locale });
+
+  return (
+    <NextIntlClientProvider locale={locale} messages={messages}>
+      {/* Add EventStructuredData component for better schema.org SEO */}
+      <EventStructuredData locale={locale} />
+      
+      {/* Script to update the HTML lang attribute */}
+      <script
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)